### PR TITLE
Update focus styles pt2

### PR DESCRIPTION
**Pull request by ricardoom**
This PR is attempting to cover issues raised in the following tickets:
Misaligned Subscription and Privacy policy links
https://jira.nypl.org/browse/WWW-389

Compile failures:
https://jira.nypl.org/browse/WWW-353

There are also numerous changes w/ respect to the mobile nav. I noticed that the `:focus` rings on  black backgrounds did not read at all, so I've made adjustments there.

If approved to merge w/ development, perhaps then we can deploy to QA as per
https://jira.nypl.org/browse/WWW-390

Thanks in advance for your studious and thorough review and general patience w/ regard to this process.

### CODE REVIEW

**Comment by ricardoom** [comment]
I need to put breaks on the review of this PR, found a couple areas of the mobile login / logout that I did not account for... please stay tuned.

**Comment by EdwinGuzman** [comment]
This looks fine to me. I cant' log in locally so I can't test that part. What was off with it?

**Comment by ricardoom** [comment]
After working on the layout for the login drop down in mobile, I became clear that the best approach would be to use flex box. but there was an extra `div` placeholder in there waiting for the logout link to be rendered once a user was logged in... 
I basically added a default style `notLoggedIn` to `display: none` initially and (hopefully 😬 ) will show up once a user is logged in... so some further testing there will be in order, but I _think_ it will work...

**Comment by EdwinGuzman** [comment]
@ricardoom this is what I see when I import it into the header app. This does not seem like it's what we want for either the focus or the layout. Let me help you tomorrow.

screen shot 2018-06-28 at 5 27 04 pm

**Comment by ricardoom** [comment]
After working on the layout for the login drop down in mobile, I became clear that the best approach would be to use flex box. but there was an extra `div` placeholder in there waiting for the logout link to be rendered once a user was logged in... 
I basically added a default style `notLoggedIn` to `display: none` initially and (hopefully 😬 ) will show up once a user is logged in... so some further testing there will be in order, but I _think_ it will work...

File: src/styles/components/_myNypl.scss [hunk 1]
```diff
@@ -210,6 +228,26 @@
...
+    // margin-right: 1px;
+  }
+
+  .researchLink {
+    flex: 1 1 40%;
+    margin: 0 1px 50px 2px;
+
+   // This is a bit confusing, but this class handles the LOGOUT link, and is visible only when logged in:
+   & + .mobileMyNypl-catalog-link {
+     borde-bottom: 2px solid #000;
```
Comment (EdwinGuzman) [inline src/styles/components/_myNypl.scss]: This should be `border`.

File: src/styles/components/_myNypl.scss [hunk 2]
```diff
@@ -210,6 +228,26 @@
     height: 25px;
     margin: 5px 0;
   }
+
+  .catalogLink {
+    border-right: solid 2px #2b2b2b;
+    flex: 1 1 40%;
+    margin: 0 1px 50px 2px;
+    // margin-left: 2px;
```
Comment (EdwinGuzman) [inline src/styles/components/_myNypl.scss]: You can remove this.

File: src/styles/components/_myNypl.scss [hunk 3]
```diff
@@ -183,21 +190,32 @@
...
+    box-shadow: 2px 2px 3px 4px rgba(100, 100, 100, 0.25);
+    display: flex;
+    flex-flow: row wrap;
   }
 
   &-greeting {
     box-sizing: content-box;
     color: #FFF;
-    text-align: left;
+    flex:1 100%;
```
Comment (EdwinGuzman) [inline src/styles/components/_myNypl.scss]: A space after the colon.

File: src/styles/components/_myNypl.scss (hunk 1 above)
Comment (ricardoom) [reply src/styles/components/_myNypl.scss]: you mean borde _isnt_ a thing?

**Comment by ricardoom** [comment]
thanks @EdwinGuzman for getting this up on dev and qa,

**Comment by ricardoom** [comment]
merging

**Pull request merged**